concepts/40 Angular Components/20 Component Configuration Syntax/37 Templates/37 Templates.md

```diff
@@ -1,4 +1,4 @@
-Templates allow you to customize UI component elements. In the following code, an [itemTemplate](/api-reference/10%20UI%20UI components/CollectionUI component/1%20Configuration/itemTemplate.md '/Documentation/ApiReference/UI_UI components/dxList/Configuration/#itemTemplate') called `listItem` and a [groupTemplate](/api-reference/10%20UI%20UI components/dxList/1%20Configuration/groupTemplate.md '/Documentation/ApiReference/UI_UI components/dxList/Configuration/#groupTemplate') called `listGroup` customize items and groups in the [List](https://js.devexpress.com/Demos/UI componentsGallery/Demo/List/GroupedList/Angular/Light). Inside the templates, the `itemData` and `groupData` variables expose item and group data objects; the `itemIndex` variable gives access to the item index.
+Templates allow you to customize UI component elements. In the following code, an [itemTemplate](/Documentation/ApiReference/UI_Components/dxList/Configuration/#itemTemplate) called `listItem` and a [groupTemplate](/Documentation/ApiReference/UI_Components/dxList/Configuration/#groupTemplate) called `listGroup` customize items and groups in the [List](/Demos/WidgetsGallery/Demo/List/GroupedList/). Inside the templates, the `itemData` and `groupData` variables expose item and group data objects; the `itemIndex` variable gives access to the item index.
```

Comment:
Links to the Widgets gallery should be full, so that it is possible to navigate to a demo when viewing our docs right on GitHub. Also, we better keep the postfix that specifies the framework because this article is about Angular.

```suggestion
Templates allow you to customize UI component elements. In the following code, an [itemTemplate](/Documentation/ApiReference/UI_Components/dxList/Configuration/#itemTemplate) called `listItem` and a [groupTemplate](/Documentation/ApiReference/UI_Components/dxList/Configuration/#groupTemplate) called `listGroup` customize items and groups in the [List](https://js.devexpress.com/Demos/WidgetsGallery/Demo/List/GroupedList/Angular/Light). Inside the templates, the `itemData` and `groupData` variables expose item and group data objects; the `itemIndex` variable gives access to the item index.
```